third-culture-software/bhima
# Add Helpdesk contact info banner

Added a new Enterprise information item:  Helpdesk Contact Info  
If this parameter is set, the contents of that field will be inserted at the bottom of the login form.

**To Test:**
- Use bhima_test
- yarn build:db
- yarn dev
- Observe at the bottom of the login  form there is nothing below the [Log in] button
- Log in and go the Admin/Enterprise page
- Set the value of the new "Helpdesk Contact Info" field (at the bottom of the left column of Enterprise info)
- Log out
- Observe the new string now shows at the bottom of the login form

NOTE:  Please check the French translation!
